specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerAppsLabelHistory.json

```diff
@@ -0,0 +1,225 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2024-10-02-preview",
+    "title": "ContainerApps API Client"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.App/containerApps/{containerAppName}/labelHistory": {
+      "get": {
+        "tags": [
+          "ContainerAppsLabelHistory"
+        ],
+        "summary": "Get the Label History for a given Container App.",
+        "operationId": "ContainerAppsLabelHistory_ListLabelHistory",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
```

Comment:
Could you move this property to CommonDefinitions? This is reference in many swagger definitions.

Reply:
moved :)

Reply:
I think theres too many places using it rn to move to ref the common defn in this PR. I can volunteer to clean up the code and have them ref the new common defn parameter in the future